source/includes/stackpath/_scripts.md

```diff
@@ -109,18 +131,92 @@ Attributes | &nbsp;
 `name`<br/>*string* | The display name of the script.
 `createdAt`<br/>*string* | Creation timestamp of the script.
 `updatedAt`<br/>*string* | The date on which the script was last updated.
-`code`<br/>*string* | Base64 encoded contents of a script.
+`code`<br/>*string* | The contents of the script.
 `version`<br/>*string* | The version number of the script.
 `routes`<br/>*Array[string]* | The routes that incoming requests should respond with a script.
 
+<!-------------------- CREATE A SCRIPT -------------------->
+
+### Create a script
+
+Create a new serverless script.
```

Comment:
A good addition would be to mention where and when it is executed

Reply:
Assuming it was not mentioned in another documentation. Maybe it is mentioned in the Retrieve script docs. Ill check

Reply:
Nevermind already taken care of!